chase/frontend/components/voting/motions.tsx

```diff
@@ -3,45 +3,40 @@ import WidgetTemplate from "../widget_template";
 import WidgetBoxTemplate from "../widget_box_template";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { Motion } from "@/custom_types";
-import { faCircleQuestion, faGavel } from "@fortawesome/free-solid-svg-icons";
+import {
+  faCircleCheck,
+  faCircleQuestion,
+  faGavel,
+  faSquarePollVertical,
+  faXmarkCircle,
+} from "@fortawesome/free-solid-svg-icons";
 import NoDataPlaceholder from "../no_data_placeholder";
 import { SmallFlag } from "../flag_templates";
 import { useI18nContext } from "@/i18n/i18n-react";
 import FlipMove from "react-flip-move";
-import { motionTestData } from "@/test_data";
 
-export default function Voting() {
-  /**
-   * This Component is used on the Voting page and displays all open motions in a list format.
-   * It also includes many animations, when a motion is added or removed.
-   * When a motion is handeled by the chair, it can be highlighted.
-   * The motions are preordered by the backend, so that the motions with most precedence are on top.
-   */
+/**
+ * This Component is used on the Voting page and displays all open motions in a list format.
+ * It also includes many animations, when a motion is added or removed.
+ * When a motion is handeled by the chair, it can be highlighted.
+ * The motions are preordered by the backend, so that the motions with most precedence are on top.
+ */
 
+export default function Motions({
+  motionData,
+  highlightedMotionId,
+  setActiveMotion,
+}: {
+  motionData: Motion[];
+  highlightedMotionId?: string;
+  setActiveMotion: (motionId: string) => void;
+}) {
   const { LL } = useI18nContext();
 
-  const [data, setData] = useState<Motion[]>(motionTestData);
-
-  useEffect(() => {
-    const pollingInterval = setInterval(() => {
-      console.log("polling");
-      setData(motionTestData);
-    }, 1000);
-    return () => clearInterval(pollingInterval);
-  }, []);
-
-  const getIcon = (personalPointOfMotion: boolean) => {
-    if (personalPointOfMotion) {
-      return faCircleQuestion;
-    } else {
-      return faGavel;
-    }
-  };
-
   return (
     <>
-      <WidgetTemplate cardTitle={LL.participants.voting.MOTIONS_HEADLINE()}>
-        {data.length === 0 ? (
+      <WidgetTemplate cardTitle="">
```

Comment:
This could be left out and the component should accept an optional param like `cardTitle?`.